db/pages/welcome_level_two_verified.rb

```diff
@@ -23,3 +22,21 @@
                   <p><a href='/'>#{I18n.t("welcome.welcome.go_to_index")}</a></p>"
   page.save!
 end
+if SiteCustomization::Page.find_by(slug: "welcome_level_two_verified").nil?
+  page = SiteCustomization::Page.new(slug: "welcome_level_two_verified", status: "published")
+  generate_content(page)
+  I18n.available_locales.each do |locale|
+    I18n.locale = locale
+    translation = page.translations.build(locale: locale)
+    generate_content(translation)
+  end
+end
+if SiteCustomization::Page.find_by(slug: "welcome_level_two_verified").nil?
```

Comment:
Was this code added twice accidentally? :thinking:

Reply:
Yes, it was. It's my first time when using git extensively and I'm quite clumsy sometimes. I messed something up and had to rebase and somehow doubled this code.

Reply:
I think that happened to all of us when we started using `git rebase` :smile:.